dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.netty.internal.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.netty.internal
+WLP-DisableAllFeatures-OnConflict: false
```

Comment:
I don't think you want this line on a new feature, it's only for old GA features

Reply:
@bjhargrave do you know the best practice for this now? feature props has a lot of stuff that I am no SME for and I am wary of not doing what most other example do.

Reply:
Sorry, I don't know anything about what `WLP-DisableAllFeatures-OnConflict` means in OL. It is not a Bnd thing. @tjwatson may be able to provide advice here.

Reply:
For new features do not set `WLP-DisableAllFeatures-OnConflict`.  This is for old features to preserve the behavior for when conflicts are found.  New features should not set this header so our build will automatically set the header value to the default (which is to force conflicts to disable all features and produce a message that the server config is not supported).  For details on the previous design issue see #13928

Reply:
Deleted